components/connection_list/index.jade

```diff
@@ -1,14 +1,12 @@
-.connection-list
-  each connection, i in connections
-    if connections.length == 1
-      include _connection
-    else
-      if i == connections.length - 2
-        include _connection
-        | &nbsp;
-        span and&nbsp;
-      else if i == connections.length - 1
-        include _connection
-      else if i < connections.length - 2
-        include _connection
-        | ,&nbsp;
\ No newline at end of file
+a.Connection(
+  data-client="Channel"
+  href=channel.href()
+  data-visibility=channel.getVisibility()
+) 
+  .Connection__inner
+    .Connection__inner__left
+      .Connection__inner__left__title= channel.get('title')
+      .Connection__inner__left__block-count &nbsp; #{channel.get('length')} blocks
+    .Connection__inner__right
+      | by&nbsp;
+      strong= channel.get('user').username
```

Comment:
OK though you can do this and be a bit less vebose: similarly I did something on the item template for the connect menu: https://github.com/aredotna/ervell/blob/master/components/connect_v2/components/item/index.jade by just wrapping a span around things.

Like how about a top level span and then the rest of things follow normal BEM? Also get rid of the "left" and "right" designations, that's a pattern that then ties DOM structure to arbitrary styling decisions.

Also try to make these templates more legible so that you can look at them and see structure. I open a lot of files and see just a big blob of really long lines with no real visual indication of what's happening. Use line breaks.